docker/puppetserver-standalone/docker-entrypoint.sh

```diff
@@ -5,6 +5,7 @@ set -e
 for f in /docker-entrypoint.d/*.sh; do
     echo "Running $f"
     chmod +x "$f"
+    sync
```

Comment:
Instead of multiple `sync` calls can we do what we've done in PDB to perform this only once - https://github.com/puppetlabs/puppetdb/pull/2887/files#diff-86cc2c2467f4688ca87a833d8bf198b2R5

Thanks!